arraysForDeletion.js

```diff
@@ -0,0 +1,246 @@
+function arraysForDeletion(){
```

Comment:
use classes ES6 with classess instead of functions. Add description to each function starting from multiline comment.